### PR TITLE
Adding crier label to test-infra.

### PR DESCRIPTION
**Pull request by michelle192837**

### CODE REVIEW

File: label_sync/labels.yaml [hunk 1]
```diff
@@ -947,6 +947,11 @@ repos:
         name: area/prow/clonerefs
         target: both
         addedBy: label
+      - color: 0052cc
+        description: Issues or PRs related to prow's crier component
+        name: area/prow-crier
```
Comment (krzyzacy) [inline label_sync/labels.yaml]: nit: prow/crier

Comment (michelle192837) [reply label_sync/labels.yaml]: Thanks!

**Comment by krzyzacy** [comment]
I think you need to update https://github.com/kubernetes/test-infra/blob/6b8c3a7f1caf29645f93a3cd3a43e959041e2442/label_sync/labels.md

**Comment by michelle192837** [comment]
Did and then forgot to push >< Whoops.

**Pull request merged**